pkg/api/types.go

```diff
@@ -130,6 +132,7 @@ func ReplicaToProtoReplica(r *Replica) *spdkrpc.Replica {
 		Rebuilding: r.Rebuilding,
 		State:      r.State,
 		ErrorMsg:   r.ErrorMsg,
+		Uuid:       r.UUID,
```

Comment:
nit: You have one commit for `sync uuid to Engine from rpc call`. It would be better if you can move the changes in `ReplicaToProtoReplica` and `ProtoReplicaToReplica` to a new commit for the consistency.